OpenTabletDriver.Configurations/Parsers/RobotPen/RobotPenTabletReport.cs

```diff
@@ -0,0 +1,34 @@
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using OpenTabletDriver.Plugin.Tablet;
+
+namespace OpenTabletDriver.Configurations.Parsers.RobotPen
+{
+    public struct RobotPenTabletReport : ITabletReport
+    {
+        public RobotPenTabletReport(byte[] report)
+        {
+            Raw = report;
+
+            Position = new Vector2
+            {
+                X = Unsafe.ReadUnaligned<ushort>(ref report[6]) | (report[7] << 8),
+                Y = Unsafe.ReadUnaligned<ushort>(ref report[8]) | (report[9] << 8)
```

Comment:
The extra fluff here is redundant.

```suggestion
                X = Unsafe.ReadUnaligned<ushort>(ref report[6]),
                Y = Unsafe.ReadUnaligned<ushort>(ref report[8])
```